modules/apps/commerce/commerce-pricing-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommercePriceModifierTest.java

```diff
@@ -332,6 +332,86 @@ public void testMultiplePercentageModifierTargetCategory()
 			modifiedPrice2.stripTrailingZeros());
 	}
 
+	@Test
+	public void testOverridePriceTargetCatalog() throws Exception {
+		frutillaRule.scenario(
+			"A type override price modifier overrides the price of a product"
+		).given(
+			"A catalog with at least one product and one price list"
+		).and(
+			"A type override price modifier targeting the product"
+		).when(
+			"The price modifier is applied to the catalog"
+		).then(
+			"The original price is overridden by the modifier"
+		);
+
+		CommerceCatalog commerceCatalog =
+			_commerceCatalogLocalService.addCommerceCatalog(
+				null, RandomTestUtil.randomString(),
+				_commerceCurrency.getCode(), LocaleUtil.US.getDisplayLanguage(),
+				_serviceContext);
+
+		CommercePriceList commercePriceList =
+			CommercePriceModifierTestUtil.addCommercePriceList(
+				commerceCatalog.getGroupId(), 0.0);
+
+		CPInstance cpInstance = CPTestUtil.addCPInstance(
+			commerceCatalog.getGroupId());
+
+		CPDefinition cpDefinition = cpInstance.getCPDefinition();
+
+		BigDecimal amount = BigDecimal.valueOf(-10);
+
+		BigDecimal price = BigDecimal.valueOf(
+			RandomTestUtil.randomDouble()
+		).setScale(
+			2, RoundingMode.valueOf(_commerceCurrency.getRoundingMode())
+		);
+
+		CommercePriceEntry commercePriceEntry = _addCommercePriceEntry(
+			"", cpDefinition.getCProductId(), cpInstance.getCPInstanceUuid(),
+			commercePriceList.getCommercePriceListId(), price);
```

Comment:
Sort it as used

Reply:
the variable defined on line 366 is used on line 374